standards_lab/ui/tests_browser.py

```diff
@@ -187,3 +198,52 @@ def test_save_schema_file(self):
 
         # click on "Save Schema" button
         self.driver.find_element_by_xpath("//button[text()='Save Schema']").click()
+
+    def test_upload_data_file(self):
+        self.get("/")
+        self.create_new_project("new_project_foo")
+
+        data_upload = self.driver.find_element_by_id("form-control-file-data")
+        data_upload.send_keys(
+            os.path.join(os.path.dirname(__file__), "fixtures", "test_data.json")
+        )
```

Comment:
Why does this not use `create_new_project_and_upload_data_file` ?

Reply:
Fixed